src/client/app/admin/authentication/user-state.service.ts

```diff
@@ -41,4 +45,11 @@ export class UserStateService {
 	public clearUser() {
 		this.user.clearUser();
 	}
+
+	private routeError(err: any): void {
+		// console.log('RouteError');
+		if (err.status === 401) {
+			this.clearUser();
```

Comment:
What's happening here is if the server responds with a 401, it assumes the user has been logged out remotely (or the session has expired), and it will clear the user and go back to the login page (or at least that's what's supposed to happen).

If the user hits a client route they aren't supposed to, they get redirected to the not authorized client page.